### ISSUE TITLE
Add a text() function

### ISSUE THREAD

**Issue by Abdur-rahmaanJ** [Abdur-rahmaanJ/ppython#6]
In template.py add a text() function taking 

```
text(<text>, coordinate-x, coordinate-y)
```
so that in snippets it can be called as

```
text("abc", 20, 20)
```

**Comment by SirGouki** [comment]
What is the usage context for this function?  I.E. is it going to print text to the screen itself?

**Comment by Abdur-rahmaanJ** [comment]
@SirGouki see template.py, a wrapper around canvas text

**Comment by pathak-deep15** [comment]
Hello @Abdur-rahmaanJ , I would like to work on this issue if its still there. Can you please describe about this function text()

**Comment by Abdur-rahmaanJ** [comment]
just like you have this
```
self.canvas.create_rectangle(x, y, x+sizeX, y+sizeY, width=self.StrokeSize_, fill=self.Fill_,
            outline=self.Stroke_)
```

in template.py

which is wrapped by this method

```
    def rect(self, x, y, sizeX, sizeY):
        self.canvas.create_rectangle(x, y, x+sizeX, y+sizeY, width=self.StrokeSize_, fill=self.Fill_,
            outline=self.Stroke_)
```

and used in a rect function

```
def rect(x, y, sizeX, sizeY):
    _p.rect(x, y, sizeX, sizeY)
```

so that in files we can call rect()

implement something similar to display texts on the canvas

**Comment by SirGouki** [comment]
I corrected my implementation to use canvas.create_text instead of adding labels.

**Comment by HassanAlsamahi** [comment]
Is this issue still going? can I work on it?

**Comment by myidispg** [comment]
I would like to work on this.

**Comment by Abdur-rahmaanJ** [comment]
Solved by PR #11
